python/lsst/ip/isr/isrStatistics.py

```diff
@@ -340,81 +345,94 @@ def measureCti(self, inputExp, overscans, gains):
         -------
         outputStats : `dict` [`str`, [`dict` [`str`, `float`]]]
             Dictionary of measurements, keyed by amplifier name and
-            statistics segment.
+            statistics segment. Everything in units based on electron.
```

Comment:
This is something that we would run in bootstrap mode for input to the PTC.  I think the gains are then 1.0 ... I'm not sure if I follow the new logic.